libraries/src/CMS/Application/ApiApplication.php

```diff
@@ -55,6 +63,11 @@ public function __construct(JInputJson $input = null, Registry $config = null, W
 		// Execute the parent constructor
 		parent::__construct($input, $config, $client, $container);
 
+		$this->triggerEvent('onGetApiFormats', [$this->formatMapper]);
```

Comment:
I wanna be sure that people don't overwrite any 'core supported' format types and map them to different formats. That's my only concern